api/tests/Behat/features-v2/admin-management/delete-others.feature

```diff
@@ -0,0 +1,33 @@
+@v2 @admin-management
+Feature: Admin - Admin users delete admin users
+
+  Scenario: A super admin user deletes other admin users
+    Given a super admin user accesses the admin app
+    And another super admin user exists
+    When I attempt to delete an existing "super admin" user
+    Then the user should be deleted
+    When I attempt to delete an existing "admin manager" user
+    Then the user should be deleted
+    When I attempt to delete an existing "admin" user
+    Then the user should be deleted
+
+  @acs
+  Scenario: An admin manager user deletes other admin users
+    Given an admin manager user accesses the admin app
+    And another admin manager user exists
+    When I attempt to delete an existing "admin manager" user
```

Comment:
i know this is super OCD overload but maybe keep the order of the checks in same order. ie super, manager, admin

Reply:
Soooo there was a reason for this and I would normally agree with you wholeheartedly but its to do with setting `$this->interactingWithUser()`. I'm doing this in the `And another admin manager user exists` step so I have the right details when I come to try to delete them. This felt a bit nicer than having  to do some kind of extra step and pass the details between them if that makes sense?

Reply:
Yep totally fine then :)